Core/ModuleInstaller.cs

```diff
@@ -255,15 +255,22 @@ public void InstallList(ICollection<CkanModule> modules, RelationshipResolverOpt
         // TODO: Return files relative to GameRoot
```

Comment:
That TODO could be solved by a simple `ksp.ToRelativeGameDir()`, does anything speak against adding it here?

Reply:
It makes for a better UI, but see my other comment, we need to update modinfo to fix the double click.